packages/fullstack/fullstack.txt

```diff
@@ -15,6 +18,7 @@ ckeditor
 classlist
 codelyzer
 collapsable
+combinatorics
```

Comment:
This term is ideally in [softwareTerms.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/master/packages/software-terms/softwareTerms.txt)